l10n_pt_account_invoicexpress/models/account_move.py

```diff
@@ -119,12 +120,23 @@ def _prepare_invoicexpress_lines(self):
             tax = line.tax_ids[:1]
             # If not tax set, force zero VAT
             tax_detail = {"name": tax.name or "IVA0", "value": tax.amount or 0.0}
+            # Because InvoiceXpress expects unit_price in EUR, check if we need to convert
+            # line currency to company currency (company should use EUR as default currency)
+            if line.currency_id.id == line.company_id.currency_id.id:
```

Comment:
Looks good.
Nitpick: you can compare record object directly and have less typing :)
```suggestion
            if line.currency_id == line.company_id.currency_id:
```

Reply:
Thanks, I thought about that but I think when we compare both objects we are comparing all the fields on those objects witch is less ideal in terms of performance. Still, I think you are right, less typing is better and I should favor this approach more moving forward.